backend/surveys/models/survey.py

```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+
+class Survey(models.Model):
+    name = models.CharField(max_length=80)
```

Comment:
You can add a description field. I think it would be nice since it can be sent to users as survey description so they know why are they getting it and and how to answer it. What do u think @cdvr1993 ?